x/lockup/keeper/lock_test.go

```diff
@@ -776,8 +778,16 @@ func (suite *KeeperTestSuite) TestEndblockerWithdrawAllMaturedLockups() {
 		suite.Require().NoError(err)
 		suite.Require().Len(locks, len(times)-i-1)
 	}
-	suite.Require().Equal(suite.App.BankKeeper.GetAccountsBalances(suite.Ctx)[1].Address, addr1.String())
-	suite.Require().Equal(suite.App.BankKeeper.GetAccountsBalances(suite.Ctx)[1].Coins, totalCoins)
+
+	// we expect that all locks with the cl token prefix do not end up in the user's account balance
```

Comment:
```suggestion
	// We expect that only non-CL locks (i.e. locks that do not have the CL token prefix) send tokens back to the user's balance when mature. This is because CL locks do not have a notion of fungible GAMM shares.
```

Reply:
Side-note: this was pretty much directly taken from my nutshell gpt diff summary :)